libsoftwarecontainer/softwarecontainer.conf.in

```diff
@@ -6,6 +6,7 @@ lxc.pts = 1
 
 @NETWORK_LXC_CONF@
 
+lxc.mount.entry = /sbin sbin none ro,bind 0 0
```

Comment:
I think this needs to be discussed more (later). /sbin contains all kinds of dangerous commands. Perhaps we could do a check in the LXC template instead to just copy iptables binary or something.